lib/dependencies/AMDDefineDependency.js

```diff
@@ -57,7 +57,7 @@ AMDDefineDependency.Template = class AMDDefineDependencyTemplate {
 			],
 			lf: [
 				"var XXX;",
-				"!(XXX = #.call(exports, __webpack_require__, exports, module))"
+				"!(XXX = #.call(exports, __webpack_require__, exports, module)); XXX === undefined && (XXX = module.exports)"
```

Comment:
Passing `exports` and `module` from the surrounding module is wrong here too.

Maybe this could fix it:

``` js
var XXX, XXX_module;
!(XXX_module = { id: YYY, exports: {}, loaded: false },
  XXX = #.call(XXX_module.exports, __webpack_require__, XXX_module.exports, XXX_module),
  XXX_module.loaded = true,
  XXX === undefined && (XXX = XXX_module.exports)
)
```